src/utils/boardImportUtils.ts

```diff
@@ -103,4 +104,25 @@ const uploadAction = (
   return promise;
 };
 
-export { openFromFileSystem, importAssetsToS3 };
+const fileMimeExtensions = {
+  "image/jpeg": [".jpg", ".jpeg"],
+  "image/png": [".png"],
+  "image/svg+xml": [".svg"],
+  "image/gif": [".gif"],
+};
+
+const allowedMimeTypes =
+  envs?.TLDRAW__ASSETS_ALLOWED_MIME_TYPES_LIST?.split(",") || [];
+
+const openAssetsFromFileSystem = async () => {
+  const extensions = (
+    allowedMimeTypes as (keyof typeof fileMimeExtensions)[]
+  ).flatMap((mimeType) => fileMimeExtensions[mimeType] || []);
```

Comment:
Please remove this casting and set fileMimeExtensions type as 
`{ [key: string]: string[] }`